app/assets/javascripts/discourse/app/components/glimmer-site-header.gjs

```diff
@@ -222,9 +222,7 @@ export default class GlimmerSiteHeader extends Component {
           }
         ).finished;
 
-        if (isTesting()) {
-          waitForPromise(animationFinished);
-        }
+        waitForPromise(animationFinished);
```

Comment:
We no longer need the `isTesting` condition?

Reply:
Yup! All waiter functions are wrapped in DEBUG checks:

https://github.com/emberjs/ember-test-waiters/blob/18acf2c0e6ea3b4d6a0ab13cec3d8047171156f4/addon/addon/wait-for-promise.ts#L35-L48

so it should't be necessary to check for DEBUG and/or isTesting at call sites